app/models/auth_source.rb

```diff
@@ -27,8 +43,8 @@ class AuthSource < ApplicationRecord
 
   validates :name, :presence => true, :uniqueness => true, :length => { :maximum => 60 }
 
-  scope :non_internal, -> { where("type NOT IN (?)", ['AuthSourceInternal', 'AuthSourceHidden']) }
-  scope :except_hidden, -> { where('type <> ?', 'AuthSourceHidden') }
+  scope :non_internal, -> { where("auth_sources.type NOT IN (?)", ['AuthSourceInternal', 'AuthSourceHidden']) }
+  scope :except_hidden, -> { where('auth_sources.type <> ?', 'AuthSourceHidden') }
```

Comment:
Just FYI, for future knowledge - you can now use `where.not(type: 'AuthSourceHidden')` or `where.not(type: ['AuthSourceInternal', 'AuthSourceHidden'])` to generate the same queries in a cleaner way

Reply:
Thanks, good to know